start.R

```diff
@@ -331,6 +331,8 @@ if (any(c("--reprepare", "--restart") %in% argv)) {
     
     # state if columns are unknown and probably will be ignored, and stop for some outdated parameters.
     source("config/default.cfg")
+    # read in GAMS related switches from main.gms
+    cfg$gms <- as.list(readSettings("main.gms"))
```

Comment:
Do we have to note somewhere that `gms` in Version `0.18.0` or higher is now needed to start REMIND?